scripts/operator-sdk.docker

```diff
@@ -13,5 +13,19 @@ VOLUME /gobase
 VOLUME /gocache
 VOLUME /gopath/src/github.com/KohlsTechnology/eunomia
 WORKDIR /gopath/src/github.com/KohlsTechnology/eunomia
-CMD echo "generating openapi..." && operator-sdk --verbose generate openapi && echo "generating k8s..." && operator-sdk --verbose generate k8s
+
+CMD \
+    go build -o /tmp/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen && \
```

Comment:
Hard code version of openapi-gen, if possible

Reply:
It's actually using the version that was previously downloaded via `go mod vendor`. This version is already locked in via `go.mod`.